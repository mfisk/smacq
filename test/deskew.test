print(ts.sql linktype) from deskew(-b 954011728 -e 954011754 -s linktype ts) from pcapfile(0325@1112-snort.pcap)
