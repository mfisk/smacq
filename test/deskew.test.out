2000-03-25 12:15:28	119
2000-03-25 12:15:28	119
2000-03-25 12:15:28	119
2000-03-25 12:15:28	119
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:28	1
2000-03-25 12:15:29	1
2000-03-25 12:15:29	1
2000-03-25 12:15:29	1
2000-03-25 12:15:29	1
2000-03-25 12:15:29	1
2000-03-25 12:15:29	1
2000-03-25 12:15:29	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:31	1
2000-03-25 12:15:34	1
2000-03-25 12:15:38	1
2000-03-25 12:15:41	1
2000-03-25 12:15:44	1
2000-03-25 12:15:47	1
2000-03-25 12:15:47	1
2000-03-25 12:15:50	1
2000-03-25 12:15:53	1
